pandas/tests/io/excel/test_readers.py

```diff
@@ -531,6 +562,10 @@ def test_bad_engine_raises(self, read_ext):
 
     @tm.network
     def test_read_from_http_url(self, read_ext):
+        if read_ext == ".xlsb":
```

Comment:
I don't think we need both skips in this test? Or do they test different things?

Reply:
This test will fail because test1.xlsb contains datetimes. However, the reason for xfailing is not correct in that case so I opted for having two xfails there so it fails for the right reason.